mozetl/shield/containers.py

```diff
@@ -0,0 +1,34 @@
+from pyspark.sql.types import StringType, LongType
+
+from ..basic import convert_pings, DataFrameConfig
+from .utils import testpilot_etl_boilerplate
+
+
+def transform_pings(sqlContext, pings):
```

Comment:
The logic change looks right, but it would be nice if we could cut down on the code duplication. It's fine if this job lives in testpilot/containers.py (even though this is a shield study).